docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -128,15 +128,17 @@ See [staging support](#staging-support) for authentication options when `dlt` co
 All write dispositions are supported.
 
 ## Data loading
-Data is loaded using `INSERT VALUES` statements by default.
+To load data into Databricks, you must set up a staging filesystem by configuring an Amazon S3 or Azure Blob Storage bucket. `parquet` is the default file format used for data uploads. As an alternative to `parquet`, you can switch to using `JSONL`.
```

Comment:
There's no need to mark Parquet and JSONL as `code` here as you're referring to file formats.

Reply:
updated